src/main/java/uk/gov/hmcts/ccd/endpoint/ui/QueryEndpoint.java

```diff
@@ -214,4 +200,23 @@ public CaseEventTrigger getEventTriggerForCase(@PathVariable("uid") String userI
                                                        eventTriggerId,
                                                        ignoreWarning);
     }
+
+    @Transactional
+    @RequestMapping(value = "/caseworkers/{uid}/jurisdictions/{jid}/case-types/{ctid}/cases/{cid}/events/{eventId}",
```

Comment:
this url looks like we are retrieving a precise event. 
Instead we are retrieving the case history associated to an event. 
Can we change it to something like:
`/caseworkers/{uid}/jurisdictions/{jid}/case-types/{ctid}/cases/{cid}/events/case-history/{eventId}` or `/caseworkers/{uid}/jurisdictions/{jid}/case-types/{ctid}/cases/{cid}/events/{eventId}/case-history`

Reply:
yeah makes sense, Stuart originally asked for the event endpoint. I will make it events/{eventId}/case-history

Reply:
done